core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -48,11 +48,14 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
       column: 'View By:',
       label: 'PERIOD',
       type: 'select',
-      options: ['1 Minute', '1 Hour', '1 Day'],
-      values: [moment(0).add(1, 'minute').valueOf() / 1000,
-        moment(0).add(1, 'hour').valueOf() / 1000,
-        moment(0).add(1, 'day').valueOf() / 1000],
-      selected: '1 Minute'
+      options: ['15 Minutes', '1 Hour', '6 Hours', '1 Day', '7 Days', '30 Days'],
+      values: [moment(0).add(15, 'minute').valueOf() / 1000,
```

Comment:
please check what values are supported for apigee vs API Gateway.

Reply:
checked